lib/sedna/core/federated_learning/federated_learning.py

```diff
@@ -30,6 +28,12 @@ class FederatedLearning(JobBase):
     """
 
     def __init__(self, estimator, aggregation="FedAvg"):
+        """
+        Initial a FederatedLearning job
+        :param estimator: Customize estimator
+        :param aggregation: aggregation algorithm for FederatedLearning
+        """
+
         protocol = Context.get_parameters("AGG_PROTOCOL", "ws")
         agg_ip = Context.get_parameters("AGG_IP", "127.0.0.1")
         agg_port = int(Context.get_parameters("AGG_PORT", "7363"))
```

Comment:
these may put in `constant.py`

Reply:
Generally, the value is obtained from the environment variable delivered by the CRD.